compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -69,7 +69,7 @@ pub struct Tables<'tcx> {
 impl<'tcx> Tables<'tcx> {
     fn rustc_ty_to_ty(&mut self, ty: Ty<'tcx>) -> TyKind {
         match ty.kind() {
-            ty::Bool => TyKind::Bool,
+            ty::Bool => stable_mir::ty::TyKind::RigidTy(stable_mir::ty::RigidTy::Bool),
```

Comment:
You can keep importing `TyKind` directly here to keep these less verbose. Also import `RigidTy` here.